extensions/adobe/experience/decisioning/decisoinitem.schema.json

```diff
@@ -0,0 +1,91 @@
+          "type": "array",
+          "description": "OLAC labels for decision item.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemIdentityMap": {
+          "$ref": "https://ns.adobe.com/xdm/context/identitymap",
+          "title": "Non person identity of the decision item.",
+          "description": "Non person identity of the decision item. It is required for ingestion in to UPS."
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemTags": {
+          "title": "Decision Item Tags",
+          "type": "array",
+          "description": "An array of Unified TagIds associated with the decision item.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemConstraints": {
+          "$ref": "https://ns.adobe.com/experience/decisioning/profile-constraint-details",
+          "title": "Decision Item Constraints",
+          "description": "Constraints define the policies that ensure a particular decision Item qualifies, is relevant and is suitable for a particular profile. Such policy rules determine the eligibility, applicability and suitability of the option. Usually the option is tested against a user profile and the events that were observed for that profile, but the qualification rule can consider any context data as long as it can be described by an XDM schema."
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemPriority": {
+          "title": "Decision Item Priority",
+          "type": "integer",
+          "description": "Priority of a decision item, based on which decision items could be ranked"
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemCatalogId": {
+          "title": "Decision Item CatalogId",
+          "type": "string",
+          "description": "Each decision item belong to a catalog. CatalogId is used to relate to the catalog. Collections are derived based on select queries and catalogId helps to avoid joins with DecisionCatalog table during query execution in hub and especially the edges."
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemCalendarConstraints": {
+          "$ref": "https://ns.adobe.com/experience/decisioning/calendar-constraint-details",
+          "title": "Decision Item Calendar Constraints",
+          "description": "Defines the date range during which this decision item is eligible. This constraint may not be strictly enforced always."
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/identitymap"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/profile-constraint-details"
```

Comment:
profile-constraint-details is a data type and is not recommended to be used here as a data type is generally used in multiple contexts within properties. This data type in particular is already used in the property above (https://ns.adobe.com/experience/decisioning/decisionitem/itemConstraints). Please remove reference to this data type from allOF